### PR TITLE
Added prototype of container up.

### PR DESCRIPTION
**Pull request by haroonf**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by panchagnula** [comment]
@haroonf can you keep this up to date. Would like to take a look at the changes next week. Have a few fit & finish recommendations. Also for local context check this out (we need to update to use the new local , global config support from CLI) https://docs.microsoft.com/en-us/cli/azure/param-persist-howto#persisted-parameter-and-global-variable-comparison

**Comment by panchagnula** [comment]
Out of curiosity can you test how this command works in interactive mode? https://docs.microsoft.com/en-us/cli/azure/interactive-azure-cli we might need to consider disabling this behavior (interactive mode) on this command if the experience is not as expected.

File: src/containerapp/azext_containerapp/custom.py [hunk 1]
```diff
@@ -1932,3 +1939,219 @@ def remove_dapr_component(cmd, resource_group_name, dapr_component_name, environ
         return r
     except Exception as e:
         handle_raw_exception(e)
+
+
+def containerapp_up(cmd,
```
Comment (panchagnula) [inline src/containerapp/azext_containerapp/custom.py:1944]: Remove container app up please.

**Comment by calvinsID** [comment]
Do we want to use a different base branch, or make this PR into Azure main branch directly? Since the `containerapp` branch is pretty old

**Comment by panchagnula** [comment]
@haroonf please rebase this to the latest & add all your changes - so that we can start an un-offiicial code review on this draft. Thanks!

**Pull request closed without merging**